File: auto-docs/executables/nodejs/hover_chart_basic.js
var plotly = require('plotly')('TestBot', 'r1neazxo9w')

var data = [
  {
    x: [0, 1, 2], 
    y: [1, 3, 2], 
    mode: "markers", 
    text: ["Text A", "Text B", "Text C"], 
    type: "scatter"
  }
];
var layout = {title: "Hover over the points to see the text"};
var graph_options = {layout: layout, filename: "hover-chart-basic", fileopt: "overwrite"}
plotly.plot(data, graph_options, function (err, msg) {
    console.log(msg);
});

File: auto-docs/executables/python/hover_chart_basic.py
import plotly.plotly as py
from plotly.graph_objs import *
py.sign_in('TestBot', 'r1neazxo9w')

data = Data([
    Scatter(
        x=[0, 1, 2],
        y=[1, 3, 2],
        mode='markers',
        text=['Text A', 'Text B', 'Text C']
    )
])
layout = Layout(
    title='Hover over the points to see the text'
)
fig = Figure(data=data, layout=layout)
plot_url = py.plot(fig, filename='hover-chart-basic', auto_open=False)